README.md

```diff
@@ -192,6 +260,30 @@ http://localhost:5601
 In order to show documents in the case-server index with Kibana, you must first create the index pattern ('Management' page) : case-server*
 
 
+### Installing / Updating docker-compose to v2
+Docker-compose v2 is necessary to have to profiles feature.  
+If possible, prefer to install it with your package manager if you are on a Unix system.
+
+> [!NOTE]  
+> You need a client (docker-cli) of ~v19~ v20 at least to have the system of cli-plugins.
+> It isn't necessary to update your Docker engine or client else.
+
+You can install docker-compose with these commands, as instructed in [the doc](https://docs.docker.com/compose/install/linux/#install-the-plugin-manually) and the [migration guide](https://docs.docker.com/compose/migrate/):
```

Comment:
```suggestion
You can install docker compose with these commands, as instructed in [the doc](https://docs.docker.com/compose/install/linux/#install-the-plugin-manually) and the [migration guide](https://docs.docker.com/compose/migrate/):
```

Reply:
It's the name of [the tool/file](https://github.com/docker/compose).

Reply:
Well the fille yes, sure, but the tool is called docker compose now, docker-compose is the previous version. We avoid confusion then.